scripts/get-job-matrix.py

```diff
@@ -72,6 +76,9 @@ def is_unit_test(pkg: str) -> bool:
         or pkg in INTEGRATION_TEST_PACKAGES
     )
 
+def is_performance_test(pkg: str) -> bool:
```

Comment:
I can't comment here, but should the return in `is_unit_test` above now be changed to

```
return not (
        pkg.startswith("github.com/pulumi/pulumi/tests")
        or pkg in INTEGRATION_TEST_PACKAGES
        or pkg in PERFORMANCE_TEST_PACKAGES
    )
```

?

Reply:
Yes, thanks!